src/cider/nrepl/middleware/track_state.clj

```diff
@@ -5,38 +5,68 @@
    [cider.nrepl.middleware.util :as util]
    [cider.nrepl.middleware.util.cljs :as cljs]
    [cider.nrepl.middleware.util.meta :as um]
-   [orchard.cljs.analysis :as cljs-ana]
    [clojure.java.io :as io]
+   [clojure.string :as string]
    [clojure.tools.namespace.find :as ns-find]
    [nrepl.misc :refer [response-for]]
    [nrepl.transport :as transport]
+   [orchard.cljs.analysis :as cljs-ana]
+   [orchard.indent :as indent]
    [orchard.java.classpath :as cp]
    [orchard.misc :as misc])
   (:import
-   (clojure.lang Namespace MultiFn)
-   java.net.SocketException
-   java.util.jar.JarFile
-   nrepl.transport.Transport))
+   (clojure.lang MultiFn Namespace)
+   (java.net SocketException)
+   (java.util.jar JarFile)
+   (nrepl.transport Transport)))
 
 (def clojure-core (try (find-ns 'clojure.core)
                        (catch Exception _e nil)))
 
 ;;; Auxiliary
 
-(defn- var-meta-with-fn
-  "Like clojure.core/meta but adds {:fn true} for functions and macros.
+(defn- inferrable-indent? [m]
```

Comment:
If we end up keeping this function (I'm not sure special handling for built-in namespaces is a good idea), let's add a docstring for it.